frontend/src/AlertsManagement/AlertsContent.js

```diff
@@ -45,30 +45,33 @@ TabPanel.propTypes = {
   value: PropTypes.any.isRequired,
 };
 
-// TODO: fetch alerts from backend and save in arrays.
-// Intended plan for reference: 
-// const allAlerts = [id1, id2, id3] where id'x' is the id of AlertX
-// const alertsMap = {id1: <Alert1>, 2: <Alert2>, 3: <Alert3>, ...};
-// const unresolvedAlerts = [0, 1] which stores the indices of the alerts in allAlerts
-// const resolvedAlerts = [2]
-const allAlerts = [0, 1, 2, 3, 4, 5, 6];
-const unresolvedAlerts = [0, 1, 2, 3];
-const resolvedAlerts = [4, 5, 6]
-
 // TODO: create constants file. ref: https://stackoverflow.com/questions/39036457/react-create-constants-file.
 const tabLabels = {
   UNRESOLVED: 0,
   RESOLVED: 1,
   ALL: 2,
 };
 
+// Helper function to create Date from Timestamp model.
+const createDate = (timestampDate) => {
+  return new Date(
+    timestampDate.date.year, timestampDate.date.month - 1, timestampDate.date.day, 0, 0, 0, 0)
```

Comment:
ooc, why is there `timestampDate.date.month - 1`?

Reply:
Javascript month is from 0 to 11

Reply:
sounds good